lib/eventsource-polyfill.js

```diff
@@ -1,5 +1,6 @@
 var EventSource = require('./eventsource')
 if (typeof window === 'object') {
   window.EventSource = window.EventSource || EventSource
+} else if (typeof global === 'object') {
+  global.EventSource = global.EventSource || EventSource
```

Comment:
Seems fine in principle, but you'll probably want to add a check to make sure `module` is a valid thing before setting it :)